Paraphrenia/Assets/Scripts/LevelEvents/RandomFlicker.cs

```diff
@@ -0,0 +1,43 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Events;
+
+public class RandomFlicker : MonoBehaviour
+{
+    public UnityEvent OnFlickerOn;
+    public UnityEvent OnFlickerOff;
```

Comment:
Unity event's moeten altijd onder alle andere variables, is een speciale uitzondering

Reply:
Also public value's zijn lowerCase